.github/workflows/pull-request.yaml

```diff
@@ -309,7 +309,13 @@ jobs:
       - name: Run policy checks on changed files
         run: |
           curl --silent --fail --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
-          https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files \
-          | opa eval --bundle build/policy/ --format values --stdin-input --fail-defined 'data.files.deny[message]'
+            -o "${{ github.event.pull_request.number }}.files.json" \
+            https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files
+          opa eval --bundle build/policy/ --format values --input "${{ github.event.pull_request.number }}.files.json" \
+            --fail-defined 'data.files.deny[message]'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Show input on failure
```

Comment:
Cool!

Reply:
I'm curious about the backstory here though :) Was there an error in your YAML file that got caught by this?

Reply:
Oh, disregard that - I saw your comments further down.